layer/composables/breacrumbs.ts

```diff
@@ -41,11 +66,18 @@ export function useBreadcrumbs() {
         // title case string regex
         let title = meta?.breadcrumbTitle || meta?.title
         if (!title) {
-          if (path === '/')
-            title = 'Home'
-          else
-          // pop last url segment and title case it
-            title = titleCase(path.split('/').pop() || '')
+          if (path === '/') {
+            title = $t ? $t(`${opts.translationPrefix}.index`) : 'Home'
```

Comment:
this will fail once deployed, Nuxt won't handle auto-imports from modules or layers

you can probably import it from `#imports`

Reply:
Same for `useI18n`

Reply:
so I tried to import it, but simply importing it when it does not exist doesn't work

any ideas ?

Reply:
is there anyway we can detect if the i18n module is installed in nuxt ? 
if so, when it is not installed, we can add our own i18n composable and log error to console when it's being used

Reply:
yes, and I did that in the last commit
@harlan-zw please check again :)